docker-compose.yaml

```diff
@@ -144,8 +144,6 @@ services:
       - agr-literature
     volumes:
       - agr-logs:/logs
-    depends_on:
```

Comment:
@sweng66 I think you don't want to remove that from the api docker-compose for main, but you did need that to run that on the devserver

Reply:
@azurebrd  got it. I changed it back! thanks